app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -70,13 +62,13 @@ class MainFragment : Fragment() {
      */
     override fun onActivityCreated(savedInstanceState: Bundle?) {
 
-
         super.onActivityCreated(savedInstanceState)
 
+        activity.let {
+            viewModel = ViewModelProvider(it!!).get(MainViewModel::class.java)
+        }
 
-        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
-
-        viewModel.firestoreService.listIncrementTime = countdownTime
```

Comment:
I changed this as I think a Service should be encapsulated from the front end fragments. Instead it should be calling a method on a ViewModel such as shown on the right -> viewModel.setIncrementTime(countdownTime).